mirgecom/simutil.py

```diff
@@ -131,10 +131,18 @@ def sim_checkpoint(discr, visualizer, eos, q, vizname, exact_soln=None,
 
         from mirgecom.io import make_rank_fname, make_par_fname
         rank_fn = make_rank_fname(basename=vizname, rank=rank, step=step, t=t)
+        if vis_timer:
+            import time
+            start = time.time()
+
         visualizer.write_parallel_vtk_file(
             comm, rank_fn, io_fields, overwrite=overwrite,
             par_manifest_filename=make_par_fname(basename=vizname, step=step, t=t))
 
+        if vis_timer:
+            stop = time.time()
+            vis_timer.add_time(stop-start)
+
```

Comment:
Use the subtimer context manager for this.

Reply:
Done.